.travis.yml

```diff
@@ -1,17 +1,27 @@
 language: python
-python:
-  - "2.7"
-#   - "3.4"  # TODO: The anaconda mro_base package seems to be broken. Re-enable when fixed.
-  - "3.5"
-  - "3.6"
+
+jobs:
+  include:
+    - os: linux
+      python: 2.7
+    - os: linux
+      python: 3.6
+    - os: linux
+      python: 3.7
+    - os: linux
+      python: 3.8
+    - os: osx
+      language: generic
+      python: 3.6
```

Comment:
Any reason we're doing only python 3.6 on OSX?

Reply:
The Mac tests kind of take a long time and I think it's ok for now to set this up for 3.6. Usually it should work on 3.7 and 3.8 as well as long as there aren't any missing pip wheel dependencies

Reply:
👍